modules/widget-visibility/widget-conditions/widget-conditions.js

```diff
@@ -163,6 +163,9 @@ jQuery( function( $ ) {
 		var $conditionsRuleMinor = $( this ),
 			$conditionsRuleMajor = $conditionsRuleMinor.siblings( 'select.conditions-rule-major' ),
 			$conditionsRuleHasChildren = $conditionsRuleMinor.siblings( 'span.conditions-rule-has-children' );
+			$condition = $conditionsRuleMinor.closest( '.condition' );
```

Comment:
Actually, wait a second. You need to change the previous line to end with a comma, not with a semicolon. Can you please fix that?